### PR TITLE
feat(networks): use a single sticky comment when posting networks checks results

### PR DESCRIPTION
**Pull request by clemsos**
# Description

To avoid cluttering the PR feed, we use a single sticky comment to post results from the network checks that get updated and moved to the end of the convo after each CI run

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/mainnet.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68 is verified
- ✅ Contract PublicLockPrevious at 0xfe9fD6af67E48D9f05Aa88679Ac294E3f28532eE is verified
- ✅ Contract Unlock at 0xe79B93f8E22676774F2A8dAd469175ebd00029FA is verified
- ✅ Contract KeyManager at 0x9A1f43090307034DBFBE2ba20320Ce815ff046D4 is verified
- ✅ Contract UDTv3 at 0x90DE74265a416e1393A450752175AED98fe11517 is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x936Ed3E71b5990bC9A94074835D08C6ca7bbFad0 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0xaF96721f8ffc136e4C170446E68Dc6744B6Ee4f4 is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0x1d8DD27432cC0aCB93B93c4486F46b67E2208359 is verified
- ✅ Unlock source code at 0xe79B93f8E22676774F2A8dAd469175ebd00029FA matches packaged UnlockV13
- ✅ PublicLock source code at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68 matches packaged PublicLockV14
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the DAO







Some additonal warnings  :

- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_PROMOCODE is missing
- ⚠️ Contract onKeyPurchaseHook_PROMO_CODE_CAPPED is missing
- ⚠️ Contract onKeyPurchaseHook_PASSWORD_CAPPED is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

**Review by julien51 — APPROVED** [review]

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/mainnet.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68 is verified
- ✅ Contract PublicLockPrevious at 0xfe9fD6af67E48D9f05Aa88679Ac294E3f28532eE is verified
- ✅ Contract Unlock at 0xe79B93f8E22676774F2A8dAd469175ebd00029FA is verified
- ✅ Contract KeyManager at 0x9A1f43090307034DBFBE2ba20320Ce815ff046D4 is verified
- ✅ Contract UDTv3 at 0x90DE74265a416e1393A450752175AED98fe11517 is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x936Ed3E71b5990bC9A94074835D08C6ca7bbFad0 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0xaF96721f8ffc136e4C170446E68Dc6744B6Ee4f4 is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0x1d8DD27432cC0aCB93B93c4486F46b67E2208359 is verified
- ✅ Unlock source code at 0xe79B93f8E22676774F2A8dAd469175ebd00029FA matches packaged UnlockV13
- ✅ PublicLock source code at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68 matches packaged PublicLockV14
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the DAO







Some additonal warnings  :

- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_PROMOCODE is missing
- ⚠️ Contract onKeyPurchaseHook_PROMO_CODE_CAPPED is missing
- ⚠️ Contract onKeyPurchaseHook_PASSWORD_CAPPED is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/mainnet.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68 is verified
- ✅ Contract PublicLockPrevious at 0xfe9fD6af67E48D9f05Aa88679Ac294E3f28532eE is verified
- ✅ Contract Unlock at 0xe79B93f8E22676774F2A8dAd469175ebd00029FA is verified
- ✅ Contract KeyManager at 0x9A1f43090307034DBFBE2ba20320Ce815ff046D4 is verified
- ✅ Contract UDTv3 at 0x90DE74265a416e1393A450752175AED98fe11517 is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x936Ed3E71b5990bC9A94074835D08C6ca7bbFad0 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0xaF96721f8ffc136e4C170446E68Dc6744B6Ee4f4 is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0x1d8DD27432cC0aCB93B93c4486F46b67E2208359 is verified
- ✅ Unlock source code at 0xe79B93f8E22676774F2A8dAd469175ebd00029FA matches packaged UnlockV13
- ✅ PublicLock source code at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68 matches packaged PublicLockV14
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the DAO







Some additonal warnings  :

- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_PROMOCODE is missing
- ⚠️ Contract onKeyPurchaseHook_PROMO_CODE_CAPPED is missing
- ⚠️ Contract onKeyPurchaseHook_PASSWORD_CAPPED is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

File: .github/workflows/_networks.yml [hunk 1]
```diff
@@ -68,6 +68,7 @@ jobs:
         with:
           message-path: results.txt
           refresh-message-position: true
+          message-id: 'network-checks-results'
```
Comment (clemsos) [inline .github/workflows/_networks.yml:71]:
```suggestion
          refresh-message-position: false
```

File: .github/workflows/_networks.yml [hunk 2]
```diff
@@ -67,7 +67,7 @@ jobs:
         uses: mshick/add-pr-comment@v2.8.2
         with:
           message-path: results.txt
-          refresh-message-position: true
+          refresh-message-position: false
```
Comment (clemsos) [inline .github/workflows/_networks.yml:70]:
```suggestion
```

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/mainnet.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68 is verified
- ✅ Contract PublicLockPrevious at 0xfe9fD6af67E48D9f05Aa88679Ac294E3f28532eE is verified
- ✅ Contract Unlock at 0xe79B93f8E22676774F2A8dAd469175ebd00029FA is verified
- ✅ Contract KeyManager at 0x9A1f43090307034DBFBE2ba20320Ce815ff046D4 is verified
- ✅ Contract UDTv3 at 0x90DE74265a416e1393A450752175AED98fe11517 is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x936Ed3E71b5990bC9A94074835D08C6ca7bbFad0 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0xaF96721f8ffc136e4C170446E68Dc6744B6Ee4f4 is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0x1d8DD27432cC0aCB93B93c4486F46b67E2208359 is verified
- ✅ Unlock source code at 0xe79B93f8E22676774F2A8dAd469175ebd00029FA matches packaged UnlockV13
- ✅ PublicLock source code at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68 matches packaged PublicLockV14
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the DAO







Some additonal warnings  :

- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_PROMOCODE is missing
- ⚠️ Contract onKeyPurchaseHook_PROMO_CODE_CAPPED is missing
- ⚠️ Contract onKeyPurchaseHook_PASSWORD_CAPPED is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/mainnet.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68 is verified
- ✅ Contract PublicLockPrevious at 0xfe9fD6af67E48D9f05Aa88679Ac294E3f28532eE is verified
- ✅ Contract Unlock at 0xe79B93f8E22676774F2A8dAd469175ebd00029FA is verified
- ✅ Contract KeyManager at 0x9A1f43090307034DBFBE2ba20320Ce815ff046D4 is verified
- ✅ Contract UDTv3 at 0x90DE74265a416e1393A450752175AED98fe11517 is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x936Ed3E71b5990bC9A94074835D08C6ca7bbFad0 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0xaF96721f8ffc136e4C170446E68Dc6744B6Ee4f4 is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0x1d8DD27432cC0aCB93B93c4486F46b67E2208359 is verified
- ✅ Unlock source code at 0xe79B93f8E22676774F2A8dAd469175ebd00029FA matches packaged UnlockV13
- ✅ PublicLock source code at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68 matches packaged PublicLockV14
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the DAO







Some additonal warnings  :

- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_PROMOCODE is missing
- ⚠️ Contract onKeyPurchaseHook_PROMO_CODE_CAPPED is missing
- ⚠️ Contract onKeyPurchaseHook_PASSWORD_CAPPED is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

File: packages/networks/src/networks/mainnet.ts [hunk 3]
```diff
@@ -175,11 +175,6 @@ export const mainnet: NetworkConfig = {
   },
 
   url: 'https://ethereum.org/en/',
```
Comment (clemsos) [inline packages/networks/src/networks/mainnet.ts:177]:
```suggestion
  url: 'https://ethereum.org/en/',
  // universalCard: {
  //   cardPurchaserAddress: '0x49814dd8a03594bE78a18455CC4Df3876ecFbD69',
  //   stripeDestinationNetwork: 'ethereum',
  //   stripeDestinationCurrency: 'usdc',
  // },
```

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/mainnet.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68 is verified
- ✅ Contract PublicLockPrevious at 0xfe9fD6af67E48D9f05Aa88679Ac294E3f28532eE is verified
- ✅ Contract Unlock at 0xe79B93f8E22676774F2A8dAd469175ebd00029FA is verified
- ✅ Contract KeyManager at 0x9A1f43090307034DBFBE2ba20320Ce815ff046D4 is verified
- ✅ Contract UDTv3 at 0x90DE74265a416e1393A450752175AED98fe11517 is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x936Ed3E71b5990bC9A94074835D08C6ca7bbFad0 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0xaF96721f8ffc136e4C170446E68Dc6744B6Ee4f4 is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0x1d8DD27432cC0aCB93B93c4486F46b67E2208359 is verified
- ✅ Unlock source code at 0xe79B93f8E22676774F2A8dAd469175ebd00029FA matches packaged UnlockV13
- ✅ PublicLock source code at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68 matches packaged PublicLockV14
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the DAO







Some additonal warnings  :

- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_PROMOCODE is missing
- ⚠️ Contract onKeyPurchaseHook_PROMO_CODE_CAPPED is missing
- ⚠️ Contract onKeyPurchaseHook_PASSWORD_CAPPED is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/mainnet.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68 is verified
- ✅ Contract PublicLockPrevious at 0xfe9fD6af67E48D9f05Aa88679Ac294E3f28532eE is verified
- ✅ Contract Unlock at 0xe79B93f8E22676774F2A8dAd469175ebd00029FA is verified
- ✅ Contract KeyManager at 0x9A1f43090307034DBFBE2ba20320Ce815ff046D4 is verified
- ✅ Contract UDTv3 at 0x90DE74265a416e1393A450752175AED98fe11517 is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x936Ed3E71b5990bC9A94074835D08C6ca7bbFad0 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0xaF96721f8ffc136e4C170446E68Dc6744B6Ee4f4 is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0x1d8DD27432cC0aCB93B93c4486F46b67E2208359 is verified
- ✅ Unlock source code at 0xe79B93f8E22676774F2A8dAd469175ebd00029FA matches packaged UnlockV13
- ✅ PublicLock source code at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68 matches packaged PublicLockV14
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the DAO







Some additonal warnings  :

- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_PROMOCODE is missing
- ⚠️ Contract onKeyPurchaseHook_PROMO_CODE_CAPPED is missing
- ⚠️ Contract onKeyPurchaseHook_PASSWORD_CAPPED is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/mainnet.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68 is verified
- ✅ Contract PublicLockPrevious at 0xfe9fD6af67E48D9f05Aa88679Ac294E3f28532eE is verified
- ✅ Contract Unlock at 0xe79B93f8E22676774F2A8dAd469175ebd00029FA is verified
- ✅ Contract KeyManager at 0x9A1f43090307034DBFBE2ba20320Ce815ff046D4 is verified
- ✅ Contract UDTv3 at 0x90DE74265a416e1393A450752175AED98fe11517 is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x936Ed3E71b5990bC9A94074835D08C6ca7bbFad0 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0xaF96721f8ffc136e4C170446E68Dc6744B6Ee4f4 is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0x1d8DD27432cC0aCB93B93c4486F46b67E2208359 is verified
- ✅ Unlock source code at 0xe79B93f8E22676774F2A8dAd469175ebd00029FA matches packaged UnlockV13
- ✅ PublicLock source code at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68 matches packaged PublicLockV14
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the DAO







Some additonal warnings  :

- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_PROMOCODE is missing
- ⚠️ Contract onKeyPurchaseHook_PROMO_CODE_CAPPED is missing
- ⚠️ Contract onKeyPurchaseHook_PASSWORD_CAPPED is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

File: .github/workflows/_networks.yml [hunk 4]
```diff
@@ -67,7 +67,6 @@ jobs:
         uses: mshick/add-pr-comment@v2.8.2
         with:
           message-path: results.txt
```
Comment (clemsos) [inline .github/workflows/_networks.yml:69]:
```suggestion
          message-path: results.txt
          message-id: networks-check
          refresh-message-position: true
```

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/mainnet.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68 is verified
- ✅ Contract PublicLockPrevious at 0xfe9fD6af67E48D9f05Aa88679Ac294E3f28532eE is verified
- ✅ Contract Unlock at 0xe79B93f8E22676774F2A8dAd469175ebd00029FA is verified
- ✅ Contract KeyManager at 0x9A1f43090307034DBFBE2ba20320Ce815ff046D4 is verified
- ✅ Contract UDTv3 at 0x90DE74265a416e1393A450752175AED98fe11517 is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x936Ed3E71b5990bC9A94074835D08C6ca7bbFad0 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0xaF96721f8ffc136e4C170446E68Dc6744B6Ee4f4 is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0x1d8DD27432cC0aCB93B93c4486F46b67E2208359 is verified
- ✅ Unlock source code at 0xe79B93f8E22676774F2A8dAd469175ebd00029FA matches packaged UnlockV13
- ✅ PublicLock source code at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68 matches packaged PublicLockV14
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the DAO







Some additonal warnings  :

- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_PROMOCODE is missing
- ⚠️ Contract onKeyPurchaseHook_PROMO_CODE_CAPPED is missing
- ⚠️ Contract onKeyPurchaseHook_PASSWORD_CAPPED is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/mainnet.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68 is verified
- ✅ Contract PublicLockPrevious at 0xfe9fD6af67E48D9f05Aa88679Ac294E3f28532eE is verified
- ✅ Contract Unlock at 0xe79B93f8E22676774F2A8dAd469175ebd00029FA is verified
- ✅ Contract KeyManager at 0x9A1f43090307034DBFBE2ba20320Ce815ff046D4 is verified
- ✅ Contract UDTv3 at 0x90DE74265a416e1393A450752175AED98fe11517 is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x936Ed3E71b5990bC9A94074835D08C6ca7bbFad0 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0xaF96721f8ffc136e4C170446E68Dc6744B6Ee4f4 is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0x1d8DD27432cC0aCB93B93c4486F46b67E2208359 is verified
- ✅ Unlock source code at 0xe79B93f8E22676774F2A8dAd469175ebd00029FA matches packaged UnlockV13
- ✅ PublicLock source code at 0xc9577b38ADA2B1b251EE99e54cC399027d547B68 matches packaged PublicLockV14
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the DAO







Some additonal warnings  :

- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_PROMOCODE is missing
- ⚠️ Contract onKeyPurchaseHook_PROMO_CODE_CAPPED is missing
- ⚠️ Contract onKeyPurchaseHook_PASSWORD_CAPPED is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

**Pull request merged**